### ISSUE TITLE
Can Multiple Datasource Be Applied For Single Flowchart

### ISSUE THREAD

**Issue by cpsarathi79** [algenty/grafana-flowcharting#128]
I would like pull data from multiple databases for assigning metrics and mappings to flowchart as I am trying to monitor the integration stuffs via flowchart. Let me know if this plugin support that and how to achieve that.

**Comment by cpsarathi79** [comment]
Thanks much for your reply :+1:

**Comment by bijwaard** [comment]
Hi Arnaud,
I use multiple datasource in metrics (flowcharting 0.8.0) and a multi-select dashboard variable to select one or more datasources. However, the rules only seem to work when I select a single datasource. I.e. when I specify the specific metric "CT1:Isum" in the rule, the rule only fires when only datasource CT1 is selected, not when 2 or more of the datasources are selected. I noticed that the suggested metric option in the rule becomes something like "{CT1,CT3}:Isum" when multiple datasources are selected, but when using that it will somehow combine the metric values from both datasources. I also tried to set the metric rule to "{CT1}:Isum" but that also does not help to get just the CT1-related metric.
![image](https://user-images.githubusercontent.com/6523124/86352743-90c90280-bc66-11ea-8981-1aa250bbfb7a.png)

Kind regards,
Dennis
